src/main/resources/default/assets/javascript/client-pagination.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+function paginate(paginationControls, update) {
+    const pagination = {
```

Comment:
Do we really want to introduce new jQuery code here?

Reply:
TODO: IE11 Support